src/main/kotlin/nl/info/zac/documentcreation/BpmnDocumentCreationService.kt

```diff
@@ -0,0 +1,133 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos, 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package nl.info.zac.documentcreation
+
+import jakarta.enterprise.context.ApplicationScoped
+import jakarta.enterprise.inject.Instance
+import jakarta.inject.Inject
+import net.atos.client.zgw.zrc.model.Zaak
+import net.atos.zac.util.MediaTypes
+import nl.info.client.smartdocuments.model.document.OutputFormat
+import nl.info.client.smartdocuments.model.document.Selection
+import nl.info.client.smartdocuments.model.document.SmartDocument
+import nl.info.client.smartdocuments.model.document.Variables
+import nl.info.client.zgw.util.extractUuid
+import nl.info.zac.app.informatieobjecten.EnkelvoudigInformatieObjectUpdateService
+import nl.info.zac.authentication.LoggedInUser
+import nl.info.zac.configuratie.ConfiguratieService
+import nl.info.zac.documentcreation.converter.DocumentCreationDataConverter
+import nl.info.zac.documentcreation.model.BpmnDocumentCreationDataAttended
+import nl.info.zac.documentcreation.model.DocumentCreationAttendedResponse
+import nl.info.zac.identity.model.getFullName
+import nl.info.zac.smartdocuments.SmartDocumentsService
+import nl.info.zac.smartdocuments.SmartDocumentsTemplatesService
+import nl.info.zac.util.AllOpen
+import nl.info.zac.util.NoArgConstructor
+import java.time.ZonedDateTime
+import java.util.UUID
+import java.util.logging.Level
+import java.util.logging.Logger
+
+@NoArgConstructor
+@ApplicationScoped
+@AllOpen
+@Suppress("LongParameterList")
+class BpmnDocumentCreationService @Inject constructor(
+    private val smartDocumentsService: SmartDocumentsService,
+    private val smartDocumentsTemplatesService: SmartDocumentsTemplatesService,
+    private val documentCreationDataConverter: DocumentCreationDataConverter,
+    private val enkelvoudigInformatieObjectUpdateService: EnkelvoudigInformatieObjectUpdateService,
+    private val configuratieService: ConfiguratieService,
+    private val loggedInUserInstance: Instance<LoggedInUser>,
+) : DocumentCreationService(
+    smartDocumentsService,
+    documentCreationDataConverter,
+    enkelvoudigInformatieObjectUpdateService,
+    configuratieService
+) {
+    companion object {
+        const val REDIRECT_METHOD = "POST"
```

Comment:
There are constants for this already no? Also this constant should not be named 'redirect method' but rather 'http post method' I would say. You use it for a redirect but the constant is just about post.

e.g. `jakarta.ws.rs.HttpMethod.POST`